modules/video/src/renderablevideoplane.cpp

```diff
@@ -27,12 +27,24 @@
 #include <openspace/documentation/documentation.h>
 #include <openspace/documentation/verifier.h>
 
+namespace {
+    // This `Renderable` creates a textured 3D plane where the texture is a video.
+    //
+    // The video can either be played back based on a given simulation time
```

Comment:
Might be nitpicky but is this clear enough that the user needs to trigger the video themselves when the video is not mapped to the simulation time? Do we want to emphasis this more to avoid further confusion?

Reply:
Hm, this is what I tried to emphasize in this description 🤔 Any suggestion for how to make it clearer? 

Maybe adding another sentence after this paragraph, saying something like this: 
"Note that, unless playback is mapped to simulation time, the video must be started manually via the user interface settings."
?

Reply:
It would also be great to create a better UI component for video playback, to make it easier to play videos. I'll create an issue for this :)

Reply:
I agree that a better UI would be great, it is a little hidden right now inside many steps in the current UI. And I like the suggested addition to make it clearer.